.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 
 * @baentsch
 /oqsprov/oqs_hyb_kem.c @bhess
-/oqs-template/generate.yml @baentsch @bhess @feventura
+/oqs-template/generate.yml @baentsch @bhess @feventura @rtjk
```

Comment:
I currently take this as your agreement to be a [committer](https://github.com/open-quantum-safe/oqs-provider/blob/main/GOVERNANCE.md#committers) in the project. Please review the link to see which obligations and permissions this (would) entail. If you do agree, please also add yourself to the corresponding README section. The addition to the GOVERNANCE.md file I'd suggest doing after some months of good cooperation: OK, @rtjk ? If you don't want that responsibility, no problem, but then the entry above needs to be removed to eliminate the error message. Please replace it by a comment with your email address in generate.yml for the Cross OIDs so we know whom to contact in case of problems.

Reply:
As per https://github.com/open-quantum-safe/liboqs/pull/1881#discussion_r1726991735 suggest to not add yourself to the CODEOWNERS file. Please add your (or your advisors') github handles by comment next to the generate.yml entries for Cross instead.

Reply:
@alexrow has [agreed](https://github.com/open-quantum-safe/liboqs/pull/1881#discussion_r1732046494) to be a committer! Holding off on the GOVERNANCE file as you suggested @baentsch.